readme.md

```diff
@@ -99,13 +99,18 @@ console.log("IO is a side effect!");
 
 ## Idempotency
 
-> A function is said to be idempotent if it has no side-effects on multiple
-executions with the the same input parameters.
+> A function is said to be idempotent if there are no additional effects of calling it a second time with the same parameters.
```

Comment:
Well, I just an other 🐜 in this universe. 

@jethrolarson You could review the file, check if we could add some more missing definitions, thanks!